### ISSUE TITLE
[bug] Color Correct (mtb) node missing

### ISSUE THREAD

**Issue by wysstartgo** [melMass/comfy_mtb#178]
## enviroment
- linux T4 gpu
- latest comfyui

## comfyui.log

```sh
matplotlib data path: /ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data
CONFIGDIR=/root/.config/matplotlib
interactive is False
platform is linux
CACHEDIR=/root/.cache/matplotlib
font search path [PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/ttf'), PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/afm'), PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/pdfcorefonts')]
generated new fontManager
[comfy_mtb] | ERROR -> Failed to parse
[comfy_mtb] | ERROR -> Failed to parse
[comfy_mtb] | INFO -> loaded 39 nodes successfuly
[comfy_mtb] | INFO -> Some nodes (15) could not be loaded. This can be ignored, but go to http://None:8188/mtb if you want more information.

## web browser
When loading the graph, the following node types were not found:
Color Correct (mtb)
Nodes that have failed to load will show as red on the graph.
```

![image](https://github.com/melMass/comfy_mtb/assets/10540192/e252535f-aac0-4d3b-8db3-6505f94ed179)

### Operating System

linux

### Comfy Mode

Comfy Portable (embed) (Default)

### Console output

```sh
matplotlib data path: /ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data
CONFIGDIR=/root/.config/matplotlib
interactive is False
platform is linux
CACHEDIR=/root/.cache/matplotlib
font search path [PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/ttf'), PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/afm'), PosixPath('/ai_workspace/ComfyUI/venv_comfyui/lib/python3.9/site-packages/matplotlib/mpl-data/fonts/pdfcorefonts')]
generated new fontManager
[comfy_mtb] | ERROR -> Failed to parse
[comfy_mtb] | ERROR -> Failed to parse
[comfy_mtb] | INFO -> loaded 39 nodes successfuly
[comfy_mtb] | INFO -> Some nodes (15) could not be loaded. This can be ignored, but go to http://None:8188/mtb if you want more information.
```

**Comment by melMass** [comment]
Can you post the full log? 
Like that I can't really tell what is failing

**Comment by wysstartgo** [comment]
> Can you post the full log? Like that I can't really tell what is failing

It looks like this is the entire startup log of my comfyui. Could you tell me how to get other logs? My current startup command is: python main.py --listen --verbose

**Comment by wysstartgo** [comment]
solved by upgrade python to 3.10,my original python version is 3.9
